scionlab/models/user_as.py

```diff
@@ -148,28 +157,21 @@ def update_attachments(self,
                 self._change_isd(isd)
 
         for att_conf in att_confs:
-            # Update attachments
             self._create_or_update_attachment(att_conf)
         for deleted_link in deleted_links:
-            # Delete links
             self._delete_attachment(deleted_link)
         for ap in aps_set:
-            # Update AttachmentPoints
             ap.split_border_routers()
             ap.trigger_deployment()
+        self._deactivate_unused_vpn_clients(att_confs)
 
-        # Deactivate useless VPNClients
-        active_vpns = set(c.attachment_point.vpn for c in att_confs if c.active and c.use_vpn)
-        for vpn_client in VPNClient.objects.filter(host=self.host):
-            if vpn_client.vpn not in active_vpns:
-                vpn_client.active = False
-                vpn_client.save()
+        self.host.update()
```

Comment:
This is a no-op, please remove it. If it's required for some weird reason, that would seem to be a bug...

Reply:
Done.